package.json

```diff
@@ -79,7 +79,7 @@
     "flow:stop": "flow stop",
     "build:lib": "node -e \"require('ncp').ncp('./src', './lib')\" && BABEL_ENV=production babel -d lib src",
     "build:browser": "webpack --config webpack.config.js",
-    "build": "npm run build:lib && npm run build:node",
+    "build": "npm run build:lib && npm run build:browser",
```

Comment:
This seems like the wrong direction to go. If you need to build for the browser before publish, change `prepublish` to `npm run build:browser` instead.